frontend/src/theme/colors.tsx

```diff
@@ -17,7 +17,7 @@ const colors = {
   },
```

Comment:
nit: I don't wanna be that one person (but I'm gonna be that one person) - can we rename all references to `colors` to `colours` 🤓

Reply:
Actually wait this will just unnecessarily add a bunch of diffs, I take this back